app/controllers/api/v1/push/subscriptions_controller.rb

```diff
@@ -21,6 +21,7 @@ def create
         endpoint: subscription_params[:endpoint],
         key_p256dh: subscription_params[:keys][:p256dh],
         key_auth: subscription_params[:keys][:auth],
+        standard: subscription_params[:standard],
```

Comment:
When omitted, `subscription_params[:standard]` is nil, which means you are building a hash with `standard: nil`. I would use something like `subscription_params[:standard] || false` or conditionally add it to the hash.

Reply:
Thanks, will push a fix